packages/core/echo/echo-pipeline/src/space/space-protocol.ts

```diff
@@ -250,7 +250,12 @@ export class SpaceProtocolSession implements WireProtocol {
     this._teleport.addExtension('dxos.mesh.teleport.blobsync', this._blobSync.createExtension());
   }
 
+  // TODO(nf): why destroy and not close?
   async destroy(): Promise<void> {
```

Comment:
yeah, rename to close

Reply:
If so then open/close (rename initialize).
But see createTeleportProtocolFactory below (if changing then change across the package).